v2/community/reusableMD/supported-tech-stacks-frontend.mdx

```diff
@@ -14,5 +17,11 @@
 </div>
 
 :::important for mobile apps
-For mobile apps, please see the "Using your own UI" section (see left nav bar)
-:::
\ No newline at end of file
+For mobile apps, please see the <RecipeLink link="custom-ui/init/frontend">**Using your own UI**</RecipeLink> section from the left nav bar
+:::
+
+:::important for nextjs 
+For **NextJS** apps, please go to the **Integrations** section from the left nav bar.
+- <RecipeLink link="nextjs/app-directory/about"> App Directory Instructions </RecipeLink>
```

Comment:
You dont need this custom component. You can directly link tot he nextjs guide like how you do in markdown. You may have to create dummy files in the community folder so that the link checker passes, but other than that, it should work fine

Reply:
You mean using a normal markdown link notation?
![image](https://github.com/user-attachments/assets/0338332e-1a49-4665-92f7-c24d11505dc1)
If so, this will not work. Because the relative paths are different:
- You are on `docs/passwordless/pre-built-ui/setup/frontend`
- You need to go to `docs/passwordless/nextjs/about`